Lesson31/Gokhan/demo-app/src/shared-components/typography/HeadingS/styles.css

```diff
@@ -0,0 +1,9 @@
+.headingS {
+  width: 76px;
+  height: 24px;
```

Comment:
It's a text element, don't set it's height. For text elements, you can use font-size and line-hight.